nodebuilder/store.go

```diff
@@ -150,6 +154,66 @@ type fsStore struct {
 	dirLock *flock.Flock // protects directory
 }
 
+// Finds a single running node and return its store path.
+// Node Type Order: Bridge, Full, Light
+// Network Order: Mainnet, Mocha, Arabica, Private
+func DiscoverOpened() (string, error) {
```

Comment:
```suggestion
// DiscoverOpened finds a path of an opened Node Store and returns its path.
// If multiple nodes are running, it only returns the path of the first found node.
// 
// Type preference order: Bridge, Full, Light
// Network preference order: Mainnet, Mocha, Arabica, Private
func DiscoverOpened() (string, error) {
```

Reply:
done, opted to favor network > node type, lmk